### PR TITLE
hello

### PR DESCRIPTION
**Pull request by ahwagner**
This commit adds a 'hello world' to your README

### CODE REVIEW

**Comment by ahwagner** [comment]
this had nothing to do with issue #1

**Review by tmooney — CHANGES_REQUESTED** [review]
Hello world should be exclamatory!

**Review by kkrysiak — CHANGES_REQUESTED** [review]
This isn't necessary

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Hello World.
+Hello World!
```
Comment (kkrysiak) [inline README.md]: Admittedly, this _is_ better

**Pull request closed without merging**